explore/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -43,26 +43,22 @@ final case class AladinContainer(
   obsConf:                ObsConfiguration,
   options:                TargetVisualOptions,
   updateMouseCoordinates: Function1[Coordinates, Callback],
-  updateFov:              Function1[Fov,
-                       Callback
-  ], // TODO Move the functionality of saving the FOV in ALadincell here
-  updateViewOffset:    Function1[Offset, Callback],
-  centerOnTarget:      View[Boolean],
-  selectedGuideStar:   Option[AgsAnalysis],
-  guideStarCandidates: List[AgsAnalysis]
+  // TODO Move the functionality of saving the FOV in ALadincell here
+  updateFov:              Function1[Fov, Callback],
+  updateViewOffset:       Function1[Offset, Callback],
+  centerOnTarget:         View[Boolean],
+  selectedGuideStar:      Option[AgsAnalysis],
+  guideStarCandidates:    List[AgsAnalysis]
 ) extends ReactFnProps[AladinContainer](AladinContainer.component)
 
 object AladinContainer {
 
-  type Props       = AladinContainer
-  type World2PixFn = Function1[Coordinates, Option[(Double, Double)]]
-  val DefaultWorld2PixFn: World2PixFn = (_: Coordinates) => None
+  type Props = AladinContainer
 
   // This is used for screen coordinates, thus it doesn't need a lot of precission
-  implicit val doubleReuse: Reusability[Double]                  = Reusability.double(1.0)
-  implicit val selectedGSReuse: Reusability[Option[AgsAnalysis]] =
-    Reusability.by(_.map(_.target.id))
-  implicit val analysisReuse: Reusability[List[AgsAnalysis]]     = Reusability.by(_.length)
+  given Reusability[Double]              = Reusability.double(1.0)
+  given Reusability[Option[AgsAnalysis]] = Reusability.by(_.map(_.target.id))
+  given Reusability[List[AgsAnalysis]]   = Reusability.by(_.length)
```

Comment:
It's so nice not to have to come up with names that are never used. 😄

Reply:
Yes, it s